apis/job/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,21 +1,19 @@
 //go:build !ignore_autogenerated
 // +build !ignore_autogenerated
 
-/*
-Copyright 2021.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
```

Comment:
This is a generated file. Question: does make generate or manifest or build change this section in anyway? If it does we may have to exclude this file from the lint process.

Reply:
you're right. I've returned the default header for these generated files and lint won't check those.